integration-tests/public/rlp-cross-contract/lib.rs

```diff
@@ -0,0 +1,67 @@
+//! A smart contract to test using RLP encoding for EVM compatibility.
+
+#![cfg_attr(not(feature = "std"), no_std, no_main)]
+
+#[ink::contract(abi_encoding = "rlp")]
```

Comment:
```suggestion
#[ink::contract(abi_encoding = "rlp")]
```

Is it only encoding or also decoding?

Reply:
It's both encoding and decoding. Do you have a recommendation? Can address in the solidity ABI PR as I have to rename from rlp anyway.

Reply:
How about just `abi`?